authority/config.go

```diff
@@ -42,6 +42,7 @@ type Config struct {
 	Address          string              `json:"address"`
 	DNSNames         []string            `json:"dnsNames"`
 	Logger           json.RawMessage     `json:"logger,omitempty"`
+	DB               json.RawMessage     `json:"db,omitempty"`
```

Comment:
Creating a new db.Config type in the `github.com/smallstep/certificates/db` might be easier than using `json.RawMessage`, and will also work better in `config.Save` method.